### PR TITLE
Support benchmarking, some marginal performance improvements

### PR DESCRIPTION
**Pull request by eschnett**

### CODE REVIEW

File: julia/output-A40/.nfs000000078000097000000f84
Comment (dstndstn) [inline julia/output-A40/.nfs000000078000097000000f84:1]: Looks like this file got included by accident!

File: julia/output-A40/.nfs000000078000098200000f85
Comment (dstndstn) [inline julia/output-A40/.nfs000000078000098200000f85:1]: Another NFS debris file

**Review by dstndstn — APPROVED** [review]
Looks good!

File: julia/output-A40/.nfs000000078000097000000f84
Comment (eschnett) [reply julia/output-A40/.nfs000000078000097000000f84:1]: Thanks. Removed.

File: julia/output-A40/.nfs000000078000098200000f85
Comment (eschnett) [reply julia/output-A40/.nfs000000078000098200000f85:1]: Thanks. Removed.

**Pull request merged**